dspace-api/src/main/java/org/dspace/workflow/WorkflowItemService.java

```diff
@@ -104,6 +104,18 @@
     public void deleteByCollection(Context context, Collection collection)
         throws SQLException, IOException, AuthorizeException;
 
+    /**
+     * Delete all workflow items submitted by by a particular e-person.
+     * FIXME: Where to trigger?
+     * @param context    The relevant DSpace Context.
+     * @throws IOException        A general class of exceptions produced by failed or interrupted I/O operations.
+     * @throws SQLException       An exception that provides information on a database access error or other errors.
+     * @throws AuthorizeException Exception indicating the current user of the context does not have permission
+     *                            to perform a particular action.
+     */
+    public void deleteWFItemWithoutSubmitter(Context context)
```

Comment:
This appears to only be used in ITs.  Since that's the case, should this be moved to the IT class where it's used?  It seems odd to create a new public method that isn't used anywhere.

Reply:
@buithaihai : Oh, I see.... there's still more to discuss/work on in this PR.  In that case, I have to push this to 8.0 (or post-7.6).  We're running short on 7.6 time and I won't have time to give feedback into this PR until after 7.6.